bin/elixir.ps1

```diff
@@ -0,0 +1,364 @@
+      $elixirParams += "+elixirc"
+      break
+    }
+
+    "--rpc-eval" {
+      $private:key = $allArgs[++$i]
+      $private:value = $allArgs[++$i]
+
+      if ($null -eq $key) {
+        Write-Error "--rpc-eval: NODE must be present"
+        exit 1
+      }
+
+      if ($null -eq $value) {
+        Write-Error "--rpc-eval: COMMAND for the '$key' node must be present"
+        exit 1
+      }
+
+      $elixirParams += "--rpc-eval"
+      $elixirParams += $key
+      $elixirParams += $value
+      break
+    }
+
+    "--boot-var" {
+      $private:key = $allArgs[++$i]
+      $private:value = $allArgs[++$i]
+
+      if ($null -eq $key) {
+        Write-Error "--boot-var: VAR must be present"
+        exit 1
+      }
+
+      if ($null -eq $value) {
+        Write-Error "--boot-var: Value for the '$key' var must be present"
+        exit 1
+      }
+
+      $elixirParams += "-boot_var"
+      $elixirParams += $key
+      $elixirParams += $value
+      break
+    }
+
+    "--pipe-to" {
+      $runErlPipe = $allArgs[++$i]
+      $runErlLog = $allArgs[++$i]
+
+      if ($null -eq $runErlPipe) {
+        Write-Error "--pipe-to: PIPEDIR must be present"
+        exit 1
+      }
+
+      if ($null -eq $runErlLog) {
+        Write-Error "--pipe-to: PIPELOG must be present"
+        exit 1
+      }
+
+      if ($runErlPipe.EndsWith("/") -or $runErlLog.EndsWith("/")) {
+        Write-Error "--pipe-to: PIPEDIR and PIPELOG must not end with a slash"
+        exit 1
+      }
+
+      break
+    }
+
+    Default {
+      $private:normalized = NormalizeArg $arg
+      $allOtherParams += $normalized
+      break
+    }
+  }
+}
+
+if ($null -eq $useIEx) {
+  $beforeExtras = @("-s", "elixir", "start_cli") + $beforeExtras
+}
+
+$beforeExtras = @("-pa", "$(Join-Path $scriptPath -ChildPath "../lib/elixir/ebin")") + $beforeExtras
+$beforeExtras = @("-noshell", "-elixir_root", "$(Join-Path $scriptPath -ChildPath "../lib")") + $beforeExtras
+
+# One MAY change ERTS_BIN= but you MUST NOT change
+# ERTS_BIN=$ERTS_BIN as it is handled by Elixir releases.
+# TODO: change when we port the releases scripts.
+# $ERTS_BIN=
+$ERTS_BIN = "$env:ERTS_BIN"
+
+$allParams = @()
+
+if ($null -ne $env:ELIXIR_ERL_OPTIONS) {
+  $private:erlFlags = $env:ELIXIR_ERL_OPTIONS -split " "
+  $allParams += $erlFlags
+}
+
+$allParams += $erlangParams
+$allParams += $beforeExtras
+$allParams += "-extra"
+$allParams += $elixirParams
+$allParams += $allOtherParams
+
+$binSuffix = ""
+
+# The variable is available after PowerShell 7.2. Previous to that, PS only worked on Windows.
+if ($isWindows -or ($null -eq $isWindows)) {
+  $binSuffix = ".exe"
+}
+
+$binPath = "$erlExec$binSuffix"
+
+if ($ERTS_BIN) {
+  $binPath = Join-Path -Path $ERTS_BIN -ChildPath $binPath
+}
+
+if ($null -eq $runErlPipe) {
+  # We double the double-quotes because they are going to be escaped by arguments parsing.
+  $paramsPart = $allParams | ForEach-Object -Process { QuoteString($_ -replace "`"", "`"`"") }
+}
+else {
```

Comment:
You know what, since we don't plan to support releases for now, I think we can remove the `--pipe-to` support for now as well as this `else` branch. Plus it only works on Unix anyway. Less code :)